cli/gen_runtime_vectors/ledger-paratime-transactions.md

```diff
@@ -0,0 +1,648 @@
+[client-sdk call]: https://github.com/oasisprotocol/oasis-sdk/blob/ea88627b4db9fc48c71a96d96a5cece08a259c6d/client-sdk/go/types/transaction.go#L251-L256
+
+## APDUSPEC changes
+
+### GET_ADDR_SR25519
+
+#### Command
+
+| Field      | Type           | Content                | Expected       |
+| ---------- | -------------- | ---------------------- | -------------- |
+| CLA        | byte (1)       | Application Identifier | 0x05           |
+| INS        | byte (1)       | Instruction ID         | 0x03           |
+| P1         | byte (1)       | Request User confirmation | No = 0      |
+| P2         | byte (1)       | Parameter 2            | ignored        |
+| L          | byte (1)       | Bytes in payload       | (depends)      |
+| Path[0]    | byte (4)       | Derivation Path Data   | 44             |
+| Path[1]    | byte (4)       | Derivation Path Data   | 474            |
+| Path[2]    | byte (4)       | Derivation Path Data   | ?              |
+| Path[3]    | byte (4)       | Derivation Path Data   | ?              |
+| Path[4]    | byte (4)       | Derivation Path Data   | ?              |
+
+First three items in the derivation path will be hardened automatically hardened
+
+#### Response
+
+| Field   | Type      | Content               | Note                     |
+| ------- | --------- | --------------------- | ------------------------ |
+| PK      | byte (32) | Public Key            |                          |
+| ADDR    | byte (??) | Bech 32 addr          |                          |
+| SW1-SW2 | byte (2)  | Return code           | see list of return codes |
+
+### GET_ADDR_SECP256K1
+
+#### Command
+
+| Field      | Type           | Content                | Expected       |
+| ---------- | -------------- | ---------------------- | -------------- |
+| CLA        | byte (1)       | Application Identifier | 0x05           |
+| INS        | byte (1)       | Instruction ID         | 0x04           |
+| P1         | byte (1)       | Request User confirmation | No = 0      |
+| P2         | byte (1)       | Parameter 2            | ignored        |
+| L          | byte (1)       | Bytes in payload       | (depends)      |
+| Path[0]    | byte (4)       | Derivation Path Data   | 44             |
+| Path[1]    | byte (4)       | Derivation Path Data   | 0              |
+| Path[2]    | byte (4)       | Derivation Path Data   | ?              |
+| Path[3]    | byte (4)       | Derivation Path Data   | ?              |
+| Path[4]    | byte (4)       | Derivation Path Data   | ?              |
+
+First three items in the derivation path will be hardened automatically hardened
+
+#### Response
+
+| Field   | Type      | Content               | Note                     |
+| ------- | --------- | --------------------- | ------------------------ |
+| PK      | byte (32) | Public Key            |                          |
+| ADDR    | byte (??) | Hex addr              |                          |
```

Comment:
Currently we use bech32-encoded string for GET_ADDR_ED25519 and I suggest we similarly use hex-encoded string here to avoid confusion. I'm not sure why we didn't use binary representation in the first place though :shrug: